athlinks-schema.ttl

```diff
@@ -14,6 +14,13 @@
 
 ### CLASSES
 
+athlinks:User
+  a rdfs:Class ;
+  rdfs:subClassOf schema:Person;
+  rdfs:comment "User .";
+  rdfs:label "User".
+  rdfs:hasProperty athlinks:id
```

Comment:
I think we should remove athlinks:User until we figure out how to store social accounts.